docs/index.rst

```diff
@@ -2,58 +2,85 @@
+   https://docs.ansible.com/ansible/latest/getting_started/basic_concepts.html#control-node
+
+
+Quickstart for Ansible Builder
+==============================
+
+To get started with Ansible Builder, you must install the ``ansible-builder`` utility and a containerization tool. Once you have the tools you need, create an :ref:`execution environment definition <builder_ee_definition>` file. By default this file is called ``execution_environment.yml``. In this file you can specify the exact content you want to include in your
+execution environment. You can specify these items:
+- the base container image
+- the version of Python
+- the version of ansible-core
+- the version of ansible-runner
+- Ansible collections, with version restrictions
+- system packages, with version restrictions
+- Python packages, with version restrictions
+- other items to download, install, or configure
+
+Choosing a base image
+---------------------
+
+You can use any base image you choose. The smaller the base image, generally, the smaller the final image. However, to make Ansible Builder more efficient, you should know what packages, if any, are already installed on the base image you use. For example, some base images already have Python installed. Others do not. If you use a base image that already has Python installed, you can omit Python in your execution environment definition file. Not all base images have package managers installed.
 
-.. _ansible-runner: https://github.com/ansible/ansible-runner
+How Ansible Builder executes
+============================
 
+Ansible Builder can execute two separate steps. The first step is to create a build instruction file (Containerfile for Podman, Dockerfile for Docker) and a build context based on the execution environment definition file. The second step is to run a containerization tool (Podman or Docker) to build an image based on the build instruction file and build context. The ``ansible-builder build`` command runs both steps. The ``ansible-builder create`` command runs only the first step. For more details, read through the :ref:`CLI usage docs <builder_cli>`.
 
-Getting Started
-***************
+How Ansible Builder builds images
+---------------------------------
 
-To get started with Ansible Builder, start by creating an
-:ref:`Execution Environment definition<Definition:Execution Environment Definition>`. This
-file allows you to specify which content you would like to include in your
-execution environment, such as collections, Python requirements, and
-system-level packages.
+Ansible Builder executes four stages when it runs your containerization tool to build a container image. The same four stages get executed if you build your container image directly with Podman or Docker, using a build instruction file and context generated by ``ansible-builder create``. These stages are:
+- Base
+- Galaxy
+- Builder
+- Final
```

Comment:
Another list not rendered properly. (Dang lists! lol)

Thanks so much for adding this, though. We really need these stages explained.